packages/google_maps_flutter/google_maps_flutter_android/example/integration_test/google_maps_tests.dart

```diff
@@ -17,6 +17,7 @@ const LatLng _kInitialMapCenter = LatLng(0, 0);
 const double _kInitialZoomLevel = 5;
 const CameraPosition _kInitialCameraPosition =
     CameraPosition(target: _kInitialMapCenter, zoom: _kInitialZoomLevel);
+const String _kCloudMapId = '8e0a97af9386fef';
```

Comment:
Not sure what we should do with the testing map Id, since it is associated with the cloud account. We might need to inject this from ci?

@stuartmorgan

Reply:
I think we do have a testing account, whose key is encrypted in https://github.com/flutter/plugins/blob/main/.cirrus.yml#L262

On iOS it is used as https://github.com/flutter/plugins/blob/edc6c0eb747fe9b8063aed896cb2d9790b2878fb/packages/google_maps_flutter/google_maps_flutter_ios/example/ios/Runner/AppDelegate.m#L15

I believe we will need to do something similar here. Maybe we should add the environment variable in a separate PR, update this PT after the environment variable is available on main.

Reply:
_kCloudMapId value does not need to be valid mapId at google maps platform. Google Maps Native SDK's silently fall back to normal map if mapId is not available. This test is meant to test the API and not the SDK behaviour itself. I'll comment this variable, and test if I can change the hash to something that is clearly random id like "0000000000000000", and **test that it still passes the tests on each platform**.

Reply:
Replaced with dummy map id `0000000000000000`, and comment `// Dummy map ID.`.